client/src/ui/organisms/article-actions/language-menu/index.scss

```diff
@@ -7,8 +7,44 @@
 }
 
 .language-menu {
+  li {
+    &:not(:first-child) {
+      padding-top: 1px;
+    }
+
+    &:not(:last-child) {
+      padding-bottom: 1px;
+    }
+  }
+
   .submenu-item {
-    padding: 0.5rem;
+    padding: 0.5rem !important;
+
+    &.locale-redirect-setting {
+      border-bottom: 1px solid var(--border-secondary) !important;
+      border-radius: 0 !important;
+      display: block;
+      font-size: var(--type-tiny-font-size);
+      padding: 0.25em 0.5em;
```

Comment:
Why have `padding: 0.25em 0.5em;` just to have it overridden by `padding: 0.5rem !important;`?